vendor/github.com/prometheus/common/expfmt/text_create.go

```diff
@@ -436,11 +436,11 @@ func writeEscapedString(w enhancedWriter, v string, includeDoubleQuote bool) (in
 func writeFloat(w enhancedWriter, f float64) (int, error) {
 	switch {
 	case f == 1:
-		return 1, w.WriteByte('1')
+		return w.WriteString("1.0")
```

Comment:
would like an ack from @kubernetes/sig-instrumentation-pr-reviews that mixing this change and version of prometheus/common with the version of prometheus/client_golang and prometheus/client_model we have is ok